src/safe_apps/tests/test_views.py

```diff
@@ -410,6 +440,7 @@ def test_provider_not_returned_in_response(self) -> None:
         self.assertCountEqual(response.json(), json_response)
```

Comment:
That's something I should fix also on the other services, but recently I found out that https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertDictEqual is the recommended way to compare dictionaries. Not applicable in this case as you are comparing a list of dictionaries, but something to be aware of 😉 . Maybe you already knew